src/Http/TracingHttpClient.php

```diff
@@ -119,6 +119,7 @@ public function request(string $method, string $url, array $options = []): Respo
 
                     $span->setAttribute(TraceAttributes::HTTP_STATUS_CODE, $info['http_code']);
                     $span->setAttribute(TraceAttributes::HTTP_URL, $info['url']);
+                    $span->addEvent('HTTP headers received');
```

Comment:
event names are namespaced and follow the same conventions as logrecord attributes, eg. `http.headers_received`
https://opentelemetry.io/docs/specs/semconv/general/events/